assets/ts/components/address-lookup/address-lookup.component.ts

```diff
@@ -97,7 +132,18 @@ class iamAddressLookup extends HTMLElement {
       });
     }
 
+    preFilledAddressBtn.addEventListener('click', (event) => {
```

Comment:
Could consider moving the common code in the 'preFilledAddressBtn' and 'switchManualBtn' event listeners, into a separate function to avoid repeating the code to add js-hide classes and loop through the inputs.

Reply:
Good idea ill make a function.